DPGAnalysis/Skims/python/ZMuSkim_cff.py

```diff
@@ -1,44 +1,166 @@
 import FWCore.ParameterSet.Config as cms
-
+from Configuration.StandardSequences.MagneticField_cff import *
```

Comment:
hi @rvenditti - the magnetic field is set at a much higher level than within a skim. Please remove this.